tests/setup/mod.rs

```diff
@@ -11,6 +11,12 @@ use xsk_rs::{
     umem::{frame::FrameDesc, CompQueue, FillQueue, Umem},
 };
 
+pub const ETHERNETPACKET: [u8; 42] = [
```

Comment:
Should have probably commented that this is just an ARP request packet pulled straight from a wireshark capture